doc/lua_api.md

```diff
@@ -9097,11 +9097,13 @@ Player properties need to be saved manually.
     hp_max = 10,
     -- Defines the maximum and default HP of the entity
     -- For Lua entities the maximum is not enforced.
-    -- For players this defaults to `minetest.PLAYER_MAX_HP_DEFAULT`.
+    -- For players this defaults to `core.PLAYER_MAX_HP_DEFAULT` (20).
+    -- `core.PLAYER_MAX_HP_DEFAULT` is defined in `builtin/game/constants.lua`
 
     breath_max = 0,
-    -- For players only. Defaults to `minetest.PLAYER_MAX_BREATH_DEFAULT`.
-
+    -- Defines the maximum amount of "breath" for the entity.
```

Comment:
"entity" should be "object" here, but that's not your mistake. The `hp_max` documentation above has the same mistake and other fields have too.

Here it's particularly problematic though, since entities don't even have "breath".

Reply:
I'll replace instances of "entity" with "object" to align better, if that works.

I wasn't sure what to call breath, except to call it breath. I thought it would be more consistent if breath_max had a similar line as hp_max describing what it does rather than assuming we know what it does.